test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.Netcore.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="FluentAssertions" Version="4.19.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
     <PackageReference Include="Microsoft.NETCore.Portable.Compatibility" Version="1.0.1" />
+    <PackageReference Include="Moq" Version="4.6.38-alpha" />

```

Comment:
>"4.6.38-alpha" /> [](start = 44, length = 17)

why do we use the "alpha" version? please use the latest RTM version

Reply:
It is the only version that is compatible with the .Netcore project.